src/Widgets/DataFormContainerWidget/DataFormContainerWidgetComponent.tsx

```diff
@@ -44,6 +44,9 @@ provideComponent(DataFormContainerWidget, ({ widget }) => {
         onSubmit={onSubmit}
         onReset={onReset}
         className={isSubmitting ? 'form-loading' : ''}
+        onClick={(e) => {
```

Comment:
* [ ] I'm surprised stopping propagation for all clicks on the form works as intended when clicking on text inside the form. However, it doesn't stop the click propagation of form input fields. So it appears to apply to buttons only. How about making the intended effect clearer by stopping the propagation on the button component?